contrib/BUILD

```diff
@@ -19,8 +19,8 @@ exports_files(["push-all.sh.tpl"])
 
 exports_files(["structure-test.sh.tpl"])
 
-exports_files(["compare_ids_test.sh.tpl"])
+exports_files(["compare_ids_test.py.tpl"])
```

Comment:
- This should be a python script, not a template
- this build file should not have an exports_files target, it should have a py_binary target(s) (e.g., https://github.com/bazelbuild/rules_docker/blob/master/container/BUILD#L67)